tests/system/aiplatform/test_dataset.py

```diff
@@ -51,6 +51,9 @@
 )
 _TEST_DATASET_DISPLAY_NAME = "permanent_50_flowers_dataset"
 _TEST_TABULAR_CLASSIFICATION_GCS_SOURCE = "gs://ucaip-sample-resources/iris_1000.csv"
+_TEST_FORECASTING_BQ_SOURCE = (
```

Comment:
I don't know if we want to add an external dependency to our system tests.

Reply:
Dataset doesn't validate anything about the data, so we could also reuse `_TEST_TABULAR_CLASSIFICATION_GCS_SOURCE`. It just might look like a bug.

Reply:
I copied the BQ data to `bq://ucaip-sample-tests:ucaip_test_us_central1`

I'm actually not sure if we should be using `gs://ucaip-sample-resources/iris_1000.csv` either to be honest.